### DISCUSSION TITLE
Images Loading after delay

### DISCUSSION THREAD

**Discussion by naman093** [vercel/next.js#66211] [Help] [answered]
### Summary

I am using nextjs and i am facing a issue that the images in the whole website is loading after delay. It is working good when i  run the project locally. But on the live website it loads the image after a delay. Can anyone tell what can be the reason or any solution to it. I have also applied lazy loading on it but still facing the issue

**Comment by serdar-webdev** [comment]
Hi folk, can you provide a code?

**Reply by naman093** [reply]
Sorry cannot do this. But i don't think even if i send the code any thing can be told from that code. Do you have any knowledge why this could happen may be due to cache problem or something like that if yes than please provide me the solution to it

**Reply by serdar-webdev** [reply]
Where the images came from? local or remote? maybe the problem is because your images size is too high?

**Reply by serdar-webdev** [reply]
Check the sizes of images in, inspect -> network tab -> assets -> images (webp,jpg,png and etc)

**Reply by naman093** [reply]
> Where the images came from? local or remote? maybe the problem is because your images size is too high?

the images are local

**Reply by naman093** [reply]
> Where the images came from? local or remote? maybe the problem is because your images size is too high?

image is of 1.3 mb

**Reply by serdar-webdev** [reply]
Whats about the sizes of images? do you use <Image/> component for images? do you use import method for local images, like
import Image1 from '/images/image-1.jpg'? which attributes you added to Image component?

**Reply by naman093** [reply]
src='/newVersionImages/abc.png'

**Reply by naman093** [reply]
the image is in public folder and i am accessing it like this `src='/newVersionImages/CTEM.png'`

**Reply by naman093** [reply]
it is not for a single picture it is for the whole website pictures. even small icons

**Reply by serdar-webdev** [reply, marked as answer]
best practices for images in next js https://image-component.nextjs.gallery/;
I suggest to you use import instead of src;
width and height attributes responsable for quality of images, i dunno how it works but it looks so;
check this out https://nextjs.org/docs/app/api-reference/components/image;
and this too https://nextjs.org/docs/app/building-your-application/optimizing/images;

**Reply by naman093** [reply]
> best practices for images in next js https://image-component.nextjs.gallery/; I suggest to you use import instead of src; width and height attributes responsable for quality of images, i dunno how it works but it looks so; check this out https://nextjs.org/docs/app/api-reference/components/image; and this too https://nextjs.org/docs/app/building-your-application/optimizing/images;

Thanks for your support i have used the next.js `<Image />` and it is resolved now. It compressed the image from 1.3 mb to 6.0 kb
